omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ObsResource1_8.java

```diff
@@ -212,7 +217,23 @@ public Model getGETModel(Representation rep) {
 		}
 		return model;
 	}
-	
+
+	private StringProperty parseAndFormatDateProperty(Object value, DateFormat isoDateFormat, DateFormat targetDateFormat) {
+		if (value instanceof String) {
```

Comment:
Unfortunately, we can't just do this in all cases, but only in cases where the Obs is of type Date.  (Not DateTime or Time or Coded or any other type).